nodejs/v12-community/health-check/booster.yaml

```diff
@@ -0,0 +1,19 @@
+name: Node.js - Health Checks
+description: Demonstrates Health Checks in Node.js
+source:
+  git:
+    url: https://github.com/nodeshift-starters/nodejs-health-check
+    ref: master
+metadata:
+  app:
+    osio:
```

Comment:
The `osio`  metadata is not used anymore and can be safely removed

Reply:
good to know